tests/06-enums.rs

```diff
@@ -21,12 +21,9 @@
 // explicit discriminant that is an integer literal. We will relax this
 // requirement in a later test case.
 //
-// Optionally if you are interested, come up with a way to support enums with a
-// number of variants that is not a power of two, but this is not necessary for
-// the test suite. Maybe there could be a #[bits = N] attribute that determines
-// the bit width of the specifier, and the getter (only for such enums) would
-// return Result<T, Unrecognized> with the raw value accessible through the
-// error type as u64:
+// Additionally, enums may support a "bits" attribute which allows to enum to
+// have a number of variants that is not a power of two. If the #[bits = N]
+// attribute is specified, like so:
```

Comment:
Btw.: Haven't noticed this during my code review but do we currently check `enum`s with `bits = N` attribute for if they have more variants to actually fit in those `N` bits? E.g.
```rust
#[derive(BitfieldSpecifier)]
#[bits = 2]
enum TwoBits { A, B, C, D, E } // 5 variants won't fit in 2 bits
```
This should cause a compilation error with a proper message ideally.

Reply:
The `09-variant-out-of-range` test should catch this.